models/model.js

```diff
@@ -1,7 +1,16 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
 
 exports.fetchCategories = () => {
   return db.query(`SELECT * FROM categories`).then((result) => {
     return result.rows;
   });
 };
+
+exports.fetchEndpoints = () => {
+  return fs.readFile("endpoints.json", "utf-8").then((result) => {
+    const jsObject = JSON.parse(result);
+    const makeJSON = JSON.stringify(jsObject);
+    return makeJSON;
+  });
```

Comment:
A model is not needed for this endpoint, this can be removed. I'd recommend reading the file in the controllers. There is no need to parse or stringify this either👍